test/integration/router-is-ready-app-gip/pages/appGip.js

```diff
@@ -0,0 +1,23 @@
+import { useRouter } from 'next/router'
```

Comment:
Can we add another page that uses `getStaticProps` to assert the behavior describe in https://github.com/vercel/next.js/pull/27473/files#r676169393 is correct

Reply:
I added some tests in 5cc4ff8. 🙇